docker-compose.yml

```diff
@@ -1,7 +1,7 @@
 version: "3"
 services:
     mongo:
-        image: mongo:4
+        image: mongo
```

Comment:
why ?

Reply:
could be ignored, my local setup doesnt work without this change